frontend/src/state/reservations.js

```diff
@@ -5,13 +5,13 @@ const {subscribe, set} = writable([])
 const reservationsStore = {
     subscribe,
     load: async () => {
-        const reservationsResult = await fetch('http://localhost:3000/reservations');
+        const reservationsResult = await fetch('/api/reservations');
```

Comment:
If you do this you may want to look into snowpack dev proxies: https://www.snowpack.dev/#dev-request-proxy

Otherwise these requests will be sent to the frontend and not the backend.

Reply:
Absolutely, the thing is that the dev proxy did not properly work. Because of that and because we didn't want to waste more time on this we left it with hard coded urls